### PR TITLE
Docs: add unusable transformations content

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,20 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+### Detect unusable transformations
+
+<!-- Kyle Cunningham -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem -->
+
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:26]: @codeincarnate - can you please add the release date for the feature here

```suggestion

<Date>

```

**Comment by codeincarnate** [comment]
@imatwawana I've added the release date and also removed the instructions for on-prem as they don't apply here. Thanks! 🙏

**Review by codeincarnate — APPROVED** [review]

**Comment by imatwawana** [comment]
Wondering if we should add something about reaching out to Support to enable the feature in Cloud?

**Comment by codeincarnate** [comment]
@imatwawana My apologies 🙏 , I totally dropped the ball here 😢. That being said I think we should be good. I believe the transformation redesign is pretty widely deployed at this point

**Comment by imatwawana** [comment]
No worries. The process this release has been a bit wild.

**Pull request merged**